public/styles.css

```
@@ -0,0 +1,194 @@
+.logo {
+    height: 3rem;
+}
+.navbar {
+    box-shadow: 0 2px 2px -2px steelblue;
+}
+.navbar-brown {
+    color: #8d3538;
+}
+.nav-link {
+    color: #8d3538;
+}
+.nav-link:hover {
+    color: steelblue;
+    cursor: pointer;
+}
+.dropdown-menu {
+    padding: 0px;
+    border: 1px solid steelblue;
+    border-bottom: none;
+}
+.dropdown-item {
+    border-bottom: 1px solid steelblue;
+}
+.dropdown-item:hover {
+    cursor: pointer;
+}
+.active-link {
+    border-bottom: 2px #8d3538 solid;
+}
+.active-item {
+    color: #8d3538;
+    background-color: #B6D4E7;
+}
+.btn-logout {
+    border-color: #8d3538;
+    color: #8d3538;
+}
+.txt-brown {
+    color: #8d3538; 
+}
+.btn-brown {
+    border-color: #8d3538;
+    color: #8d3538;
+}
+.btn-brown:hover {
+    color: steelblue;
+    transform: scale(1.05);
+}
+a {
+    color: black;
+}
+.btn-logout:hover {
+    color: steelblue;
+    transform: scale(1.05);
+}
+.table-heading {
+    background-color: #8e3436;
+    height: 30px;
+    color:#ffff;
+}
+.table thead > tr > th {
+    vertical-align: bottom;
+    border-bottom: 2px solid #8e3436;
+}
+table > tbody > tr > td{
+    border-bottom: 1px solid rgb(0,0,0, 0.2);
+}
+.coding-notstarted{
```

```suggestion
.coding-notstarted {
```
and all other cases like this, I'm not going to manually highlight them all. Your code editor can probably be configured to help out here

---

okay, thanks